recipe/meta.yaml

```diff
@@ -46,7 +46,7 @@ about:
 
   description: |
     Multidicts are useful for working with HTTP headers, URL query args etc.
-  doc_url: https://multidict.readthedocs.io/
```

Comment:
I would remove `en/stable` where possible so that we don't prevent internationalization.